instrumentation/testing/testing.go

```diff
@@ -154,6 +161,57 @@ func (test *Test) End() {
 	test.span.Finish()
 }
 
+func (test *Test) extractTestLoggerOutput() {
+	output := extractTestOutput(test.t)
+	if output == nil {
+		return
+	}
+	outStr := string(*output)
+	var logArray []map[string]interface{}
+	for _, match := range TESTING_LOG_REGEX.FindAllString(outStr, -1) {
+		matches := TESTING_LOG_REGEX.FindStringSubmatch(match)
+
+		if matches[1] == "" {
+			msg := matches[3]
+			if strings.Index(msg, "    ") == 0 {
+				msg = msg[4:]
+			}
+			if len(logArray) > 0 {
+				logItem := logArray[len(logArray)-1]
+				logItem["message"] = logItem["message"].(string) + "\n" + msg
+			}
```

Comment:
Can't this logic be embedded within `TESTING_LOG_REGEX` somehow? Seems a bit hackish.

Reply:
done